aries-test-harness/features/0037-present-proof.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Aries agent present proof functions RFC 0037 
+
+   @T001-API10-RFC0037 @P1 @AcceptanceTest @NeedsReview @wip
+   Scenario Outline: Present a proof to a verifier where the verifier has requested the proof
+      Given we have "3" agents
+         | name  | role     |
+         | Bob   | Holder   | 
+         | Faber | Issuer   |
+         | Acme  | Verifier |
+      And “Bob” has a credential issued from "Faber"
+      And “Bob and Acme” have an existing connection
+      And "Acme" sends a presentation request to "Bob" 
```

Comment:
Looks good. I believe this should be a "When"? We should always try to utilize the full gherkin syntax. Given does the preconditions and pre-steps before we get to the steps that are the actions that will result in the outcomes we want. The When are those actions. "Then" is used to show outcome of those When steps.